source/Plugins/HumbleLibrary/HumbleLibrary.cs

```diff
@@ -132,7 +132,11 @@ public override IEnumerable<Game> ImportGames()
                                 {
                                     var exst = allTpks.FirstOrDefault(a =>
                                     !a.human_name.IsNullOrEmpty() &&
-                                    (a.human_name == product.human_name || Regex.IsMatch(a.human_name, product.human_name + @".+\sKey$")));
+                                    (a.human_name.Equals(product.human_name, StringComparison.OrdinalIgnoreCase) || 
+                                    Regex.IsMatch(a.human_name, product.human_name + @".+\sKey$", RegexOptions.IgnoreCase) || 
+                                    Regex.IsMatch(a.human_name, product.human_name + @".*\s\(?Steam\)?$", RegexOptions.IgnoreCase) ||
+                                    Regex.IsMatch(a.human_name + @"\s*+$", product.human_name, RegexOptions.IgnoreCase)));

```

Comment:
What is this last check for?

Reply:
That's checking for the problem I had with Assault Android Cactus. For that game, a.human_name == "Assault Android Cactus", while product.human_name == "Assault Android Cactus+". The regex checks if there are any number of spaces and then a "+" symbol, just in case.